lib/bibliothecary/parsers/haskell.rb

```diff
@@ -0,0 +1,32 @@
+require "json"
+
+module Bibliothecary
+  module Parsers
+    class Haskell
+      include Bibliothecary::Analyser
+
+      def self.mapping
+        {
+          /.*\.cabal$/ => {
+            kind: 'lockfile',
```

Comment:
this should be `'manifest'`, lockfiles are where the full list of transitive dependencies and all resolved versions are stored, which looks to be like `cabal freeze` generates one called `cabal.config`

Reply:
I wasn't sure about this one.  Thanks for correcting it.